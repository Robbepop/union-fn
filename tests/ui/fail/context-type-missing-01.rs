fn main() {}

#[union_fn::union_fn]
trait Foo {
    type Context;

    fn foo(ctx: &mut Self::Context) {}
}
