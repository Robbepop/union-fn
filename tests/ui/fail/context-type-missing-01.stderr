error: must have a default for Context type in #[union_fn] trait
 --> tests/ui/fail/context-type-missing-01.rs:5:5
  |
5 |     type Context;
  |     ^^^^^^^^^^^^^
